### ISSUE TITLE
Issue with receiving mp3s

### ISSUE THREAD

**Issue by sabret00the** [DrKLO/Telegram#1095]
If I send an mp3 to a friend's phone. The tracks aren't coming up in their media library and when the tracks are manually played, they have both the artist and title set as null. Emailing the same tracks doesn't present the same problem.

**Comment by AUDI-A4** [comment]
because files dont have any proper name.It is happening with me also

**Comment by Mindstormer619** [comment]
Yeah, same problem as #1029

**Comment by AUDI-A4** [comment]
issue is resolved in latest update.just tap onto the file then click to save to downloads.File will be saved in original name

**Comment by Mindstormer619** [comment]
@AUDI-A4 No, it still saves under that encoded name. I tried it with a PDF file right now. Latest update, too.

**Comment by AUDI-A4** [comment]
it seems to me you are right.i have tried too and was saved into encoded filename not original name

**Comment by sabret00the** [comment]
I don't think it's a filename issue. It would appear that either the file is being clipped or edited by Telegram. Hopefully @DrKLO can look into it.
